File: webservice/Frikollection_Api/Frikollection_Api/DTOs/User/UserDto.cs
using Frikollection_Api.DTOs.Collection;
using Frikollection_Api.DTOs.Notification;

namespace Frikollection_Api.DTOs.User
{
    public class UserDto
    {
        public Guid UserId { get; set; }
        public string Avatar { get; set; }
        public string Nickname { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateOnly? Birthday { get; set; }
        public string? Biography { get; set; }
        public ICollection<CollectionDto> OwnCollections { get; set; } = new List<CollectionDto>();
    }
}

File: mobile_desktop_app/FrikollectionMobileDesktop/composeApp/src/commonMain/kotlin/org/example/frikollection_mobile_desktop/models/user/UserDto.kt
package org.example.frikollection_mobile_desktop.models.user

import kotlinx.serialization.Serializable
import org.example.frikollection_mobile_desktop.models.collection.CollectionDto

@Serializable
data class UserDto(
    val userId: String,
    val avatar: String,
    val nickname: String,
    val firstName: String,
    val lastName: String,
    val birthday: String? = null,
    val biography: String? = null,
    val ownCollections: List<CollectionDto> = emptyList()
)